index.js

```diff
@@ -4,6 +4,7 @@ import ContentSlider from "content-slider";
 import VideoController from "video-controller";
 import TabsPanel from "tabs-panel";
 import FilterBar from "filter-bar";
+import { salarySurveyInit } from 'compare-salary';
```

Comment:
My biggest concern here is that I think, semantically, all this JS belongs in the aquent/vitamintalent.com repo instead, since it's dealing more with site functionality rather than pattern display. I'll let you decide if you want to make that change now. Obviously At Some Point we'll be recreating this thing and can deal with bigger decisions then.